### PR TITLE
Invoice - add Invoice Line and clear required fields

### PR DESCRIPTION
**Pull request by GabrielBlondeau-Kirby**
I add new fields: accounting_reference, external_reference, and invoice_lines

Remove a few required filds.

### CODE REVIEW

File: source/includes/_invoices.md [hunk 1]
```diff
@@ -2,22 +2,39 @@
...
+payments | `array` | List of [`Payment`](#payments) objects associated with invoice
+invoice_lines | `array` | List of `Invoice Line` associate with an invoice
+accounting_reference | `string` | Is the ID reference from an Accounting system. If your company is using an Account system supported by ShareActor this field will be automatically setup.
+external_reference | `string` | You can set this field to put your own invoice reference or if you are using a third party.
 
+### Invoice Line
+
+Attributes | Type | Description
+---------- | ---- | ------
+**amount** | `number` | Amount as a number with two decimals. eg., 12.34.
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: Invoice line amount. Number with two decimals. eg., 12.34.

File: source/includes/_invoices.md [hunk 2]
```diff
@@ -2,22 +2,39 @@
...
+**currency** | `string` | ISO 4217 code of currency, for example: "EUR"
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: ISO 4217 currency code. For example "EUR"

Please change above also 👍

File: source/includes/_invoices.md [hunk 3]
```diff
@@ -2,22 +2,39 @@
...
+**description** | `string` | The description of what the invoice line is. e.g, a shipping line.
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: Description of the invoice line. e.g, a shipping line.

File: source/includes/_invoices.md [hunk 4]
```diff
@@ -2,22 +2,39 @@
...
+external_reference | `string` | If you are using a 3rd party the `external_reference` could be the ID returns by it.
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: If you are using third-party services, the `external_reference` could be their ID

File: source/includes/_invoices.md [hunk 5]
```diff
@@ -2,22 +2,39 @@
...
+product | `object` | [`Product`](#products) ID to be associated withe the invoice line
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: [`Product`](#products) associated with the invoice line

File: source/includes/_invoices.md [hunk 6]
```diff
@@ -2,22 +2,39 @@
...
+quantity | `number` | Usually it's use to set the quantity of [`Product`](#products) _default is 1_
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: Usually? 

Or is it: The quantity of [`Product`](#products) in the invoice line _default is 1_

File: source/includes/_invoices.md [hunk 7]
```diff
@@ -2,22 +2,39 @@
 
 Invoices allow you to store invoice information for later payment.
 
+The Invoice integrates a system of **Invoice Line**. An invoice line is defined by an `amount`, a `currency` and a `description`.
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: This is not the best English... 

Maybe better with: The invoice has invoice lines (Maybe a link to the description here). An invoice line consists mainly of `amount`, a `currency` and a `description`. See description for more details.

File: source/includes/_invoices.md [hunk 8]
```diff
@@ -2,22 +2,39 @@
...
 
 Attributes | Type | Description
 ---------- | ---- | ------
-**user** | `object` | [`User`](#users) associated with invoice
-**amount** | `number` | The invoice amount. Two decimals.
-**currency** | `string` | ISO 4217 code of currency, for example: "EUR"
-**account_number** | `string` | Account number of the issuer that will receive the payment.
+user | `object` | [`User`](#users) associated with invoice
+amount | `number` | The invoice amount. Two decimals.
+currency | `string` | ISO 4217 code of currency, for example: "EUR"
```
Comment (KristopherJordan) [inline source/includes/_invoices.md]: see comment under in invoice lines

**Comment by GabrielBlondeau-Kirby** [comment]
Close this PR because of https://github.com/shareactorIO/slate/pull/65.

**Pull request closed without merging**